modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -257,6 +259,21 @@ public String getSearchInputName() {
 		return _searchInputName;
 	}
 
+	public String getSearchResultsTitle() {
+		if (_searchResultsTitle == null) {
+			HttpServletRequest httpServletRequest = getRequest();
+
+			ThemeDisplay themeDisplay =
+				(ThemeDisplay)httpServletRequest.getAttribute(
+					WebKeys.THEME_DISPLAY);
+
+			_searchResultsTitle = LanguageUtil.get(
+				themeDisplay.getLocale(), "search-results");
```

Comment:
This is a bit of a smell, because we are changing `_searchResultsTitle` that should stay immutable and represent value set though API. It doesn't make much of difference in this case, but there are tags that have multiple places that reference the same internal value.

Cleaner code here would be
```java
return LanguageUtil.get(
	themeDisplay.getLocale(), "search-results");
```

Reply:
ok! updating that too